fastplotlib/graphics/_base.py

```diff
@@ -450,7 +492,7 @@ def graphics(self) -> np.ndarray[Graphic]:
         """The Graphics within this collection. Always returns a proxy to the Graphics."""
         if self._graphics_changed:
```

Comment:
what was the reason for having this bool?

Reply:
prevent remaking the array of proxies which can take a few millisecond